timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -391,3 +415,343 @@ def _get_model_type(self):
+        Number of CPU cores used to fit the models in parallel.
+        When set to a float between 0.0 and 1.0, that fraction of available CPU cores is used.
+        When set to a positive integer, that many cores are used.
+        When set to -1, all CPU cores are used.
+    max_ts_length : int, default = 2500
+        If not None, only the last ``max_ts_length`` time steps of each time series will be used to train the model.
+        This significantly speeds up fitting and usually leads to no change in accuracy.
+    """
+
+    def _get_model_type(self):
+        from statsforecast.models import CrostonClassic
+
+        return CrostonClassic
+
+
+class IMAPAModel(AbstractStatsForecastIntermittentDemandModel):
+    """Intermittent demand forecasting model using the Intermittent Multiple Aggregation Prediction Algorithm
+    [Petropoulos2015]_.
+
+
+    Based on `statsforecast.models.IMAPA <https://nixtla.mintlify.app/statsforecast/docs/models/imapa.html>`_.
+
+
+    References
+    ----------
+    .. [Petropoulos2015] Petropoulos, Fotios, and Nikolaos Kourentzes. "Forecast combinations for intermittent
+        demand." Journal of the Operational Research Society 66.6 (2015): 914-924.
+
+
+    Other Parameters
+    ----------------
+    n_jobs : int or float, default = 0.5
+        Number of CPU cores used to fit the models in parallel.
+        When set to a float between 0.0 and 1.0, that fraction of available CPU cores is used.
+        When set to a positive integer, that many cores are used.
+        When set to -1, all CPU cores are used.
+    max_ts_length : int, default = 2500
+        If not None, only the last ``max_ts_length`` time steps of each time series will be used to train the model.
+        This significantly speeds up fitting and usually leads to no change in accuracy.
+    """
+
+    def _get_model_type(self):
+        from statsforecast.models import IMAPA
+
+        return IMAPA
+
+
+class ConformalizedZeroModel(AbstractStatsForecastIntermittentDemandModel):
+    """A naive forecaster that always returns 0 forecasts across the prediction horizon, where the prediction
+    intervals are computed using conformal prediction.
+
+    Other Parameters
+    ----------------
+    n_jobs : int or float, default = 0.5
+        Number of CPU cores used to fit the models in parallel.
+        When set to a float between 0.0 and 1.0, that fraction of available CPU cores is used.
+        When set to a positive integer, that many cores are used.
+        When set to -1, all CPU cores are used.
+    max_ts_length : int, default = 2500
+        If not None, only the last ``max_ts_length`` time steps of each time series will be used to train the model.
+        This significantly speeds up fitting and usually leads to no change in accuracy.
+    """
+
+    def _get_model_type(self):
+        raise NotImplementedError
+
+    def _get_local_model(self, local_model_args: Dict):
+        raise NotImplementedError
```

Comment:
Why not just remove these methods?

Reply:
hmm :/ explicit is better than implicit? I'll remove `_get_local_model` and also comment what I'm trying to say in `_get_model_type`